CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Version 1.7.1
```

Comment:
There is already a 1.7.1, so this will have to be 1.7.2.

for 1.7.1, I think @Pessimistress bumped mapbox version but it didn't make it to the changelog, maybe you can add it when you update this PR?